collect_app/src/main/res/values/colors.xml

```diff
@@ -10,6 +10,7 @@
     <color name="colorSurface">#FFFFFF</color>
     <color name="colorSurfaceVariant">#dce3e9</color>
     <color name="colorOnSurface">#000000</color>
+    <color name="colorSurfaceContainerLow">#F7F2FA</color>
```

Comment:
I think there's a typo here. That should be `#F7F7FA`.

Reply:
This https://m3.material.io/components/buttons/specs#0eea2a85-b4d7-4c74-b08e-98410b9412c7 says the container color should be `Surface container low` and it's value in https://m3.material.io/styles/color/the-color-system/tokens?preview=true#7fd4440e-986d-443f-8b3a-4933bff16646 is `#F7F2FA`

Reply:
Right, but that's the color for the default Material theme where the surface color is a very light pink (`#FEF7FF`). Our surface color is just a plain white (`#FFFFFF`) so we tend to use greys for our darker variants. If you look at the main menu buttons again with the `#F7F2FA` color, you'll see they're now a pink shade.